tests/test_utils.py

```diff
@@ -625,3 +654,99 @@ def test_csv_latin_1_encoding(self):
+        from natcap.invest import utils
+
+        # Willamette valley in lat/lon for reference
+        lon = -124.525
+        lat = 44.525
+
+        base_srs = osr.SpatialReference()
+        base_srs.ImportFromEPSG(4326)  # WSG84 EPSG
+
+        target_srs = osr.SpatialReference()
+        target_srs.ImportFromEPSG(4326)
+
+        transformer = utils.create_coordinate_transformer(base_srs, target_srs)
+        x, y, _ = transformer.TransformPoint(lon, lat)
+
+        expected_x = -124.525
+        expected_y = 44.525
+
+        self.assertEqual(expected_x, x)
+        self.assertEqual(expected_y, y)
+
+    def test_latlon_to_projected_transformer(self):
+        """Utils: test transformer for lat/lon to projected."""
+        from natcap.invest import utils
+
+        # Willamette valley in lat/lon for reference
+        lon = -124.525
+        lat = 44.525
+
+        base_srs = osr.SpatialReference()
+        base_srs.ImportFromEPSG(4326)  # WSG84 EPSG
+
+        target_srs = osr.SpatialReference()
+        target_srs.ImportFromEPSG(26910)  # UTM10N EPSG
+
+        transformer = utils.create_coordinate_transformer(base_srs, target_srs)
+        x, y, _ = transformer.TransformPoint(lon, lat)
+
+        expected_x = 378816.2531852932
+        expected_y = 4931317.807472325
+
+        self.assertEqual(expected_x, x)
+        self.assertEqual(expected_y, y)
+
+    def test_projected_to_latlon_transformer(self):
+        """Utils: test transformer for projected to lat/lon."""
+        from natcap.invest import utils
+
+        # Willamette valley in lat/lon for reference
+        known_x = 378816.2531852932
+        known_y = 4931317.807472325
+
+        base_srs = osr.SpatialReference()
+        base_srs.ImportFromEPSG(26910)  # UTM10N EPSG
+
+        target_srs = osr.SpatialReference()
+        target_srs.ImportFromEPSG(4326)  # WSG84 EPSG
+
+        transformer = utils.create_coordinate_transformer(base_srs, target_srs)
+        x, y, _ = transformer.TransformPoint(known_x, known_y)
+
+        expected_x = -124.52500000000002
+        expected_y = 44.525
+
+        self.assertAlmostEqual(expected_x, x, places=3)
+        self.assertAlmostEqual(expected_y, y, places=3)
+
+    def test_projected_to_projected_transformer(self):
+        """Utils: test transformer for projected to projected."""
+        from natcap.invest import utils
+
+        # Willamette valley in lat/lon for reference
+        known_x = 378816.2531852932
+        known_y = 4931317.807472325
+
+        base_srs = osr.SpatialReference()
+        base_srs.ImportFromEPSG(26910)  # UTM10N EPSG
+
+        target_srs = osr.SpatialReference()
+        target_srs.ImportFromEPSG(26910)  # UTM10N EPSG
+
+        transformer = utils.create_coordinate_transformer(base_srs, target_srs)
+        x, y, _ = transformer.TransformPoint(known_x, known_y)
+
+        expected_x = 378816.2531852932
+        expected_y = 4931317.807472325

```

Comment:
Same comment as above, but now that I see all the tests, I see how it's nice to keep a consistent pattern. Could define `expected_x = known_x`, though?

Reply:
Hey @davemfish , I'm not sure what you were looking for here but I made some changes here 484b2a1b7c33fa2445149cd01b9e90cc85ce4512 that might address the issue?

Reply:
Hey @dcdenu4 , I just meant that these same exact numbers are being assigned to two different variables in this test because the input coords are expected to be identical to the output coords. No big deal though.